notes/randomNote/gkeep_api_service.py

```diff
@@ -0,0 +1,15 @@
+import gkeepapi
+import random
+
+
+class gkeepApiService:
```

Comment:
```suggestion
class GKeepApiService:
```
all class names should be in camel case